doc/notebooks/.ipynb_checkpoints/GSoC2021_GRASS_Jupyter_Integration-checkpoint.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Delete this file and then if you feel like you can manage some more Git today, find the `.gitignore` in the source code root directory and add `.ipynb_checkpoints` directory there.